test/raven.test.js

```diff
@@ -2342,13 +2335,13 @@ describe('Raven (public API)', function() {
         }
       });
       assert.throw(function() {
-        fn.__raven__;
+        return fn.__raven__;
```

Comment:
I had to add `return`, because otherwise Babel would remove this line (unassigned expression statement).